scripts/protoDC2/protoDC2_valid_obsids.py

```diff
@@ -0,0 +1,66 @@
+import pickle
+import numpy as np
+import pandas as pd
+from lsst.sims.catUtils.utils import ObservationMetaDataGenerator
+
+def fov_overlaps_protoDC2(x, y, half_size=2.5, radius=1.77):
```

Comment:
Should this method us `lsst.sims.utils.angularSeparation` to properly take spherical geometry into account?

(Though I guess it technically doesn't matter for protoDC2, since it is centered at Dec=0)